studio/components/interfaces/BillingV2/Usage/Bandwidth.tsx

```diff
@@ -0,0 +1,189 @@
+import clsx from 'clsx'
+import ShimmeringLoader from 'components/ui/ShimmeringLoader'
+import SparkBar from 'components/ui/SparkBar'
+import { useDailyStatsQuery } from 'data/analytics/daily-stats-query'
+import { useProjectSubscriptionQuery } from 'data/subscriptions/project-subscription-query'
+import {
+  ProjectUsageResponse,
+  UsageMetric,
+  useProjectUsageQuery,
+} from 'data/usage/project-usage-query'
+import dayjs from 'dayjs'
+import { PRICING_TIER_PRODUCT_IDS, USAGE_APPROACHING_THRESHOLD } from 'lib/constants'
+import { formatBytes } from 'lib/helpers'
+import Link from 'next/link'
+import { Button, IconAlertTriangle } from 'ui'
+import BarChart from './BarChart'
+import SectionContent from './SectionContent'
+import SectionHeader from './SectionHeader'
+import { USAGE_CATEGORIES } from './Usage.constants'
+
+export interface BandwidthProps {
+  projectRef: string
+}
+
+const Bandwidth = ({ projectRef }: BandwidthProps) => {
+  const { data: usage } = useProjectUsageQuery({ projectRef })
+  const { data: subscription } = useProjectSubscriptionQuery({ projectRef })
+  const { current_period_start, current_period_end } = subscription?.billing ?? {}
+  const startDate = new Date((current_period_start ?? 0) * 1000).toISOString()
+  const endDate = new Date((current_period_end ?? 0) * 1000).toISOString()
+  const categoryMeta = USAGE_CATEGORIES.find((category) => category.key === 'bandwidth')
+
+  const upgradeUrl =
+    subscription?.tier.supabase_prod_id === PRICING_TIER_PRODUCT_IDS.ENTERPRISE
```

Comment:
Are we missing the upgrade to Pro here?

Reply:
actually good point here - originally if the project is on pro or payg, the url will link them to the plan selection page
but we should just bring them to the plan management page (where they can choose addons and stuff)